tests/qconvolutional_test.py

```diff
@@ -155,6 +155,9 @@ def test_qconv1d():
           x)
   model = Model(inputs=x, outputs=y)
 
+  # Print qstats to make sure it works with Conv1D layer
+  print_qstats(model) 
```

Comment:
Could you add a test function that 

something like "assert number_of_operations (of Conv1D) == expected_value" in a network that only contains Conv1D. 

So the next developers who change your code may not pass the test if there is mistake.

Does it make sense? thanks!